src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -29,21 +29,24 @@ export function useHeatmapSize<T>(
   const availableWidth = width - leftAxisWidth;
   const availableHeight = height - bottomAxisHeight;
 
-  // Determine how to compute mesh size to fit available space while maintaining aspect ratio
+  // Determine how to compute canvas size to fit available space while maintaining aspect ratio
   const shouldAdjustWidth = availableWidth >= availableHeight * aspectRatio;
 
-  const meshWidth = shouldAdjustWidth
+  const canvasWidth = shouldAdjustWidth
     ? availableHeight * aspectRatio
     : availableWidth;
-  const meshHeight = shouldAdjustWidth
+  const canvasHeight = shouldAdjustWidth
```

Comment:
I've renamed the variables here to make it clear that the styles are computed based on the size of the canvas.